ehitchhiking-ui/src/app/shared/components/trip/trip.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { Trip } from "../trips-modal/trips";
+
+@Component({
+  selector: 'app-trip',
+  templateUrl: './trip.component.html',
+  styleUrls: ['./trip.component.sass']
+})
+export class TripComponent implements OnInit {
+
+  @Input() trip: Trip;
+
+  @Output() change: EventEmitter<Trip>;
+
+  constructor() {
+    this.change = new EventEmitter();
+  }
+
+  ngOnInit() {
+  }
+
+  makeFavorite() {
+     this.trip.isFavorite = !this.trip.isFavorite;
+    }
+    //this.change.emit(this.trip);
```

Comment:
pls remove commented out code and empty linrs

Reply:
okay